### PR TITLE
docs : Add Prisma Editor

### PR DESCRIPTION
**Pull request by mohammed-bahumaish**
Prisma editor is a powerful tool to visualize and edit Prisma Schema. You can easily create, modify and maintain complex database structures and be sure that your databases are organized and consistent.

With Prisma Editor, you can take advantage of a variety of incredible features, including real-time visualization, a user-friendly interface, SQL generation, and even the ability to edit your schema directly from the graph. But that's not all! We have also integrated the power of OpenAI's natural language processing API, which enables you to generate boilerplate schema code using natural language prompts.

Closes #

## 🎯 Changes

What changes are made in this PR? Is it a feature or a bug fix?

## ✅ Checklist

- [ ] I have followed the steps listed in the [Contributing guide](https://github.com/trpc/trpc/blob/main/CONTRIBUTING.md).
- [ ] If necessary, I have added documentation related to the changes made.
- [ ] I have added or updated the tests related to the changes made.

### CODE REVIEW

**Review by KATT — COMMENTED** [review]
Looks cool but not directly correlated to trpc afaict?

**Comment by mohammed-bahumaish** [comment]
> Looks cool but not directly related to trpc afaict?

The editor uses tRPC, so I added it to (Open-source projects using tRPC) list.

**Comment by mohammed-bahumaish** [comment]
@KATT Please consider merging. let me know if there is anything that i should do. Thanks.

**Comment by ObaidQatan** [comment]
This looks great for me, will definitely boost working with prisma!

**Review by KATT — DISMISSED** [review]
Ohhh thanks, I misunderstood! Awesome project :)

**Comment by KATT** [comment]
Lint needs fixin

**Comment by mohammed-bahumaish** [comment]
> Lint needs fixin

Thanks for approving, I will fix Lint now.

**Comment by mohammed-bahumaish** [comment]
> Lint needs fixin

It's fixed :)

File: www/docs/main/awesome-trpc.mdx [hunk 1]
```diff
@@ -85,14 +85,15 @@ A collection of resources on tRPC.
...
+| [SST](https://sst.dev) - A framework that makes it easy to build serverless apps.                                | https://github.com/serverless-stack/sst             |
+| [Beam](https://planetscale.com/blog/introducing-beam) - A simple message board for your organization or project. | https://github.com/planetscale/beam                 |
+| [Rallly](https://rallly.co) - Self-hostable doodle poll alternative.                                             | https://github.com/lukevella/rallly                 |
+| [Hilde](http://hilde.gg) - Match-making app for games like foosball, air hockey and similar                      | https://github.com/nehalist/hilde                   |
+| [Answer Overflow](https://www.answeroverflow.com/) - Discord bot that indexes help channel content into Google   | https://github.com/AnswerOverflow/AnswerOverflow    |
+| [Prisma Editor](https://prisma-editor.up.railway.app) - powerful tool to visualize and edit Prisma Schema.       | https://github.com/mohammed-bahumaish/prisma-editor |
```
Comment (mozzius) [inline www/docs/main/awesome-trpc.mdx:96]:
```suggestion
| [Prisma Editor](https://prisma-editor.up.railway.app) - Powerful tool to visualize and edit Prisma Schema       | https://github.com/mohammed-bahumaish/prisma-editor |
```

Comment (mozzius) [reply www/docs/main/awesome-trpc.mdx:96]: (this may require lint fixes again, sorry!)

**Review by KATT — APPROVED** [review]

**Pull request merged**